itunesCall.js

```diff
@@ -1,8 +1,8 @@
 const fetch = require("node-fetch");
 
-const getPodcastData = async () => {
+const getAllData = async (activity) => {
     try {
-		const response = await fetch(`https://itunes.apple.com/search?term=$podcast`, {
+		const response = await fetch(`https://itunes.apple.com/search?term=${activity}`, {
```

Comment:
I appreciate how you've made this dynamic! Made it very easy to get the music set up when we were working together on this!

Reply:
Thank you :)